src/main/java/com/technocredits/orangeHRM/base/PredefinedMethods.java

```diff
@@ -49,18 +49,10 @@ public void click(String locator) {
 	 * checkURL method check if the login is done successfully, based on comparing
```

Comment:
Change method name in comment.

Reply:
method name changed to checkURL in Comment